onboarding/src/Components/AddUserTable/AddUserTablePage.js

```diff
@@ -34,23 +30,17 @@ function AddUserTablePage(props) {
 
 
     return(
-        <div className="app">
-            <header className="app-header app-header-dark">
-                <Navbar loggedUser={ loggedUser } />
-            </header>
-            <LeftMenu packageId = { packageIdRef.current } loggedUser={ loggedUser } />
-            <main className="app-main">
-                <div className="wrapper">
-                    <div className="page">
-                        <div className="page-inner">
-                            <PageAddressBar page = { "Wyślij pracownikowi" } loggedUser={ loggedUser } />
-                            <AddUserTable loggedUser={ loggedUser } packageId={ packageIdRef.current } packageCurrent={ packageObj } showModal={ popUpConfirmationModal } />
-                        </div>
+        <main className="app-main">
+            <div className="wrapper">
```

Comment:
These parent nodes seems to be obsolete because they are in `App.js` also
resulting
```
<main className="app-main">
    <div className="wrapper">
        <main className="app-main">
            <div className="wrapper">
```